{"name": "Открытие раздела 'Медиа'", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <tests.test_ui.TestKinopoiskUI object at 0x000002471BC71940>\n\n    @allure.story(\"Навигация + обход капчи\")\n    @allure.title(\"Открытие раздела 'Медиа'\")\n    def test_open_media(self):\n        with allure.step(\"Открытие главной страницы\"):\n            self.main_page.open()\n        with allure.step(\"Установка куки авторизации для обхода капчи\"):\n            self.main_page.add_auth_cookie()\n        with allure.step(\"Переход по ссылке 'Медиа'\"):\n>           self.main_page.open_media()\n\ntests\\test_ui.py:61: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\main_page.py:84: in open_media\n    self.click(media_link)\npages\\base_page.py:34: in click\n    element = self.wait.until(EC.element_to_be_clickable(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"8ca2134e-d6c1-4aed-972a-27fbb1b5dffe\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x000002471BD1B920>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Wait until the method returns a value that is not False.\n    \n        Calls the method provided with the driver as an argument until the\n        return value does not evaluate to ``False``.\n    \n        Parameters:\n        -----------\n        method: callable(WebDriver)\n            - A callable object that takes a WebDriver instance as an argument.\n    \n        message: str\n            - Optional message for :exc:`TimeoutException`\n    \n        Return:\n        -------\n        object: T\n            - The result of the last call to `method`\n    \n        Raises:\n        -------\n        TimeoutException\n            - If 'method' does not return a truthy value within the WebDriverWait\n            object's timeout\n    \n        Example:\n        --------\n        >>> from selenium.webdriver.common.by import By\n        >>> from selenium.webdriver.support.ui import WebDriverWait\n        >>> from selenium.webdriver.support import expected_conditions as EC\n    \n        # Wait until an element is visible on the page\n        >>> wait = WebDriverWait(driver, 10)\n        >>> element = wait.until(EC.visibility_of_element_located((By.ID, \"exampleId\")))\n        >>> print(element.text)\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\nC:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:146: TimeoutException"}, "steps": [{"name": "Открытие главной страницы", "status": "passed", "steps": [{"name": "Открытие страницы ''", "status": "passed", "parameters": [{"name": "url", "value": "''"}], "start": 1744545897826, "stop": 1744545898538}], "start": 1744545897826, "stop": 1744545898538}, {"name": "Установка куки авторизации для обхода капчи", "status": "passed", "steps": [{"name": "Установка куки авторизации", "status": "passed", "steps": [{"name": "Ожидание появления элемента ('tag name', 'body')", "status": "passed", "parameters": [{"name": "locator", "value": "('tag name', 'body')"}], "start": 1744545898972, "stop": 1744545899003}], "start": 1744545898538, "stop": 1744545899003}], "start": 1744545898538, "stop": 1744545899003}, {"name": "Переход по ссылке 'Медиа'", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"C:\\Users\\ASUS\\Desktop\\HW\\SkyPro\\coursework\\Kurs5\\kurs5\\kurs5\\kinopoisk_autotests\\tests\\test_ui.py\", line 61, in test_open_media\n    self.main_page.open_media()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\ASUS\\Desktop\\HW\\SkyPro\\coursework\\Kurs5\\kurs5\\kurs5\\kinopoisk_autotests\\pages\\main_page.py\", line 84, in open_media\n    self.click(media_link)\n    ~~~~~~~~~~^^^^^^^^^^^^\n  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\ASUS\\Desktop\\HW\\SkyPro\\coursework\\Kurs5\\kurs5\\kurs5\\kinopoisk_autotests\\pages\\base_page.py\", line 34, in click\n    element = self.wait.until(EC.element_to_be_clickable(locator))\n  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 146, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Переход на страницу 'Медиа'", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\ASUS\\Desktop\\HW\\SkyPro\\coursework\\Kurs5\\kurs5\\kurs5\\kinopoisk_autotests\\pages\\main_page.py\", line 84, in open_media\n    self.click(media_link)\n    ~~~~~~~~~~^^^^^^^^^^^^\n  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\ASUS\\Desktop\\HW\\SkyPro\\coursework\\Kurs5\\kurs5\\kurs5\\kinopoisk_autotests\\pages\\base_page.py\", line 34, in click\n    element = self.wait.until(EC.element_to_be_clickable(locator))\n  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 146, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Клик по элементу ('xpath', \"//a[contains(text(), 'Медиа')]\")", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\ASUS\\Desktop\\HW\\SkyPro\\coursework\\Kurs5\\kurs5\\kurs5\\kinopoisk_autotests\\pages\\base_page.py\", line 34, in click\n    element = self.wait.until(EC.element_to_be_clickable(locator))\n  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 146, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "parameters": [{"name": "locator", "value": "('xpath', \"//a[contains(text(), 'Медиа')]\")"}], "start": 1744545899003, "stop": 1744545919169}], "start": 1744545899003, "stop": 1744545919169}], "start": 1744545899003, "stop": 1744545919170}], "start": 1744545897826, "stop": 1744545919170, "uuid": "7c5b5a73-56cf-4b0a-929a-22058b5bf2c7", "historyId": "26faaba3857eb0a3676d10332164994a", "testCaseId": "26faaba3857eb0a3676d10332164994a", "fullName": "tests.test_ui.TestKinopoiskUI#test_open_media", "labels": [{"name": "story", "value": "Навигация + обход капчи"}, {"name": "feature", "value": "UI Тесты Кинopoиска"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ui"}, {"name": "subSuite", "value": "TestKinopoiskUI"}, {"name": "host", "value": "DESKTOP-QEBLV6A"}, {"name": "thread", "value": "8820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ui"}]}